### PR TITLE
1.6.2-beta2

### PR DESCRIPTION
**Pull request by outofambit**

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.6.2-beta2": [
+      "[Added] Add manual conflict resolution options in merge conflicts modal - #6062",
+      "[Added] Wire up Syntax Highlighting for Crystal and Julia - #6710. Thanks @KennethSweezy!",
```
Comment (shiftkey) [inline changelog.json]: Following the pattern from previous releases:

```suggestion
      "[Added] Crystal and Julia syntax highlighting - #6710. Thanks @KennethSweezy!",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.6.2-beta2": [
+      "[Added] Add manual conflict resolution options in merge conflicts modal - #6062",
```
Comment (shiftkey) [inline changelog.json]: Tentative feels on this - in the `1.5.0` changelog we didn't say whether it was a dialog or a modal. If have settled on a name for this somewhere else let me know.

```suggestion
      "[Added] Add manual conflict resolution options in merge conflicts dialog - #6062",
```

Comment (billygriffin) [reply changelog.json]: Thanks for the suggestion @shiftkey! I think I'm going to make a slightly different suggestion that doesn't specify the place.

Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Allow users to also resolve manual conflicts when resolving merge conflicts - #6062",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.6.2-beta2": [
+      "[Added] Allow users to also resolve manual conflicts when resolving merge conflicts - #6062",
+      "[Added] Crystal and Julia syntax highlighting - #6710. Thanks @KennethSweezy!",
+      "[Fixed] Fix pointy corners with border-radius - #6635. Thanks @lisavogtsf!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Fix pointy corners on commit message text area - #6635. Thanks @lisavogtsf!",
```

**Pull request merged**